From b2cf10cb8df0d6c9b66547f8b98b73f9055c7376 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Nikola=20Forr=C3=B3?= <nforro@redhat.com>
Date: Wed, 22 Nov 2017 14:42:33 +0100
Subject: [PATCH] Remove Easter egg entirely

Six years is a reasonable shelf life for a joke, but I think its time
has passed now.

* src/man.c (main): Remove Easter egg.
---
 src/man.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/man.c b/src/man.c
index 5a72e1f..20d5a5e 100644
--- a/src/man.c
+++ b/src/man.c
@@ -1158,13 +1158,6 @@ int main (int argc, char *argv[])
 	debug ("\nusing %s as pager\n", pager);
 
 	if (first_arg == argc) {
-		/* http://twitter.com/#!/marnanel/status/132280557190119424 */
-		time_t now = time (NULL);
-		struct tm *localnow = localtime (&now);
-		if (localnow &&
-		    localnow->tm_hour == 0 && localnow->tm_min == 30)
-			fprintf (stderr, "gimme gimme gimme\n");
-
 		if (print_where) {
 			manp = get_manpath ("");
 			printf ("%s\n", manp);
-- 
2.13.6

